app/models/manageiq/providers/google/inventory/parser.rb

```diff
@@ -671,16 +671,8 @@ def load_balancer_health_check_members(persister_lb_health_check, target_pool)
 
       # Lookup our health state in the health status map; default to
       # "OutOfService" if we can't find a mapping.
-      status = "OutOfService"
-      unless instance_health.nil?
-        gcp_status = instance_health[0][:health_state]
-
-        if GCP_HEALTH_STATUS_MAP.include?(gcp_status)
-          status = GCP_HEALTH_STATUS_MAP[gcp_status]
-        else
-          _log.warn("Unable to find an explicit health status mapping for state: #{gcp_status} - defaulting to 'OutOfService'")
```

Comment:
Not sure if this log is very helpful and was able to clean the code up a lot if I didn't include it